multi-node/aws/README.md

```diff
@@ -30,7 +30,7 @@ export AWS_SECRET_ACCESS_KEY=MY-SECRET-KEY
 Creating a KMS key can be done via the AWS web console or via the AWS cli tool.
 
 ```shell
-$ aws kms --region=<your-region> create-key --description="kube-aws assets"
+$ aws kms --region=us-west-1 create-key --description="kube-aws assets"
```

Comment:
We explicitly use us-west-1 below in the `kube-aws init` example, and I felt like it was better to shift both to something concrete and working than make the init example use another `<variable>` - (in this case being concrete has a particular value, because the mapping between "US West (N. California)" and "us-west-1" isn't all that obvious to noobs like me)